src/components/RoundForm.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <div>
+    <h2>Input Rounds</h2>
+    <p>{{ msg }}</p>
+    
+    <input v-model="date" type="text" size="35" placeholder="Enter date, yyyy-mm-dd"><br><br>
+    
+    <select v-model="club" @change="changeClub">
+      <option :value="{}" default selected disabled>Select club</option>
+      <option v-for="clubOpt in clubOptions" :key="clubOpt.name"
+      :value="clubOpt">
+        {{ clubOpt.name }}
+      </option>
+    </select><br><br>
+
+    <table>
+      <tr v-for="(course, i) in courses" :key="course.name">
+        <td>{{ course.name }}</td>
+        <td>
+          <button @click="removeCourse(i)">Remove</button>
+        </td>
+      </tr>
+      <tr v-if="canAddCourses">
+        <td>
+          <select v-model="newCourse">
+            <option :value="{}" default selected disabled>Select course</option>
+            <option v-for="course in newCourseOptions" :key="course.name"
+            :value="course">
+              {{ course.name }}
+            </option>
+          </select>
+        </td>
+        <td>
+          <button @click="addCourse"
+          :disabled="Object.keys(newCourse).length == 0">
+            Add
+          </button>
+        </td>
+      </tr>
+    </table><br>
+
+    <div v-for="(course, courseIndex) in courses" :key="course.name">
+      <h4>Course - {{ course.name }} </h4>
+      
+      <select v-model="course.tees">
+        <option value="" default selected disabled>Select tees</option>
+        <option v-for="tees in course.teesOptions" :key="tees">
+          {{ tees }}
+        </option>
+      </select><br><br>
+
+      <input v-model.number="course.numHoles" number type="number" 
```

Comment:
Maybe bc I've never played golf, it wasn't super intuitive that I had to add the total number of holes I'm going to play/have played in this field...perhaps some help text?

Reply:
Agreed, I am setting `placeholder="# holes#` in the line below but it's not working, I think it's because `course.numHoles` gets initialized to 0 and this overwrites the placeholder. I will look into it since it's not the best UX

Reply:
Added!

<img width="1280" alt="Screen Shot 2020-11-30 at 9 19 01 PM" src="https://user-images.githubusercontent.com/8866443/100689123-d7c03e80-3351-11eb-85ea-0c724b6c1f2b.png">